SECURITY.md

```diff
@@ -15,13 +15,15 @@ libcurl, report it on [HackerOne](https://hackerone.com/curl).
 
 We treat security issues with confidentiality until controlled and disclosed responsibly.
 
-## OpenSSF Scorecard
-
-curl has earned Gold status on the OpenSSF Best Practices, reflecting its adherence to
-rigorous security and best practice standards. This achievement highlights curl's
-comprehensive documentation, secure development processes, effective change control
-mechanisms, and strong maintenance routines. Meeting these criteria demonstrates curl's
-commitment to security and reliability, ensuring the project's sustainability and
-trustworthiness. This recognition by OpenSSF underscores curl's role as a leader in
-open-source software practices. More information can be found on
-their [OpenSSF page](https://www.bestpractices.dev/projects/63).
+## OpenSSF Best Practices
+
+curl has achieved Gold status on the Open Source Security Foundation (OpenSSF)
+[Best Practices](https://bestpractices.dev/) (formerly Core Infrastructure
+Initiative [CII] Best Practices), reflecting its adherence to rigorous
```

Comment:
```suggestion
Initiative Best Practices), reflecting its adherence to rigorous
```

Simplifies it a little and saves us from having to whitelist `CII` as a word for the spellchecker.

Reply:
The only reason I put CII in there was because (weirdly) OpenSSF Scorecard still refers to 'CII Best Practices' rather than 'OpenSSF Best Practices', so the CII term is still undead.

I'm cool with the proposed change though, and I totally understand why putting CII into the spell check might be a bad idea because it's an easy typo for CI.

Reply:
That's exactly my reasoning - and I really don't want to introduce a special "allow-list" for this document alone. A small sacrifice but not too bad I think.